doc/project/checklist.md

```diff
@@ -436,6 +443,12 @@ In addition, the fully implemented functional coverage plan, the observed covera
 
 The V3 checklist has been reviewed to understand the scope and estimate effort.
 
+## V2S
+
+### SEC_CM_VERIFIED
+
+Security countermeasures are verified.
```

Comment:
We need to add TODO here - eventually we will describe our cm verif methodology in our overall methodology doc. This item should point to it.